### ISSUE TITLE
Fix Banner Image in IndividualStories and IndividualResources

### ISSUE THREAD

**Issue by seanmnguyen** [CS-Social-Good-CalPoly/Vera#299]
### Problem
Currently, there’s a missing image with the alt text reading “logo” on every `IndividualStories` or `IndividualResources` page. To see this, click either the `Stories` or `Resources` page, then select any of the stories or resources, respectively.

This is because these pages use a `Banner` component at the top, but due to the way it's been implemented, the `Banner` is missing some props/information for an image.

### Task
- [ ] Explore the codebase, paying particular attention to the `Banner` component, where it's used, and how those relevant pages are routed/connected together
- [ ] Remove the "logo" alt text. Some approaches could be implementing a condition within the component that only includes the logo image when appropriate, creating a new component entirely, or anything else you think of!

### Notes
Make sure to create a branch for this task and run `git pull` each time before you start working.

<img width="1512" alt="Image" src="https://github.com/user-attachments/assets/595e5550-2ce7-43a4-a174-7534e29e1f45" />
